test/test_jit_tracing.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+from common import TEST_DEVICES
+
+import torch
+import torchgeometry as tgm
+import torch.nn as nn
+
+
+class WarpPerspective(nn.Module):
+
+    def __init__(self, output_size):
+        super(WarpPerspective, self).__init__()
+
+        self.output_height, self.output_width = output_size
+
+    def forward(self, x, M):
+
+        x = tgm.warp_perspective(x, M,
+                                 dsize=(self.output_height,
+                                        self.output_width))
+
+        return x
+
+
+class WarpAffine(nn.Module):
+
+    def __init__(self, output_size):
+        super(WarpAffine, self).__init__()
+
+        self.output_height, self.output_width = output_size
+
+    def forward(self, x, M):
+
+        x = tgm.warp_affine(x, M,
+                            dsize=(self.output_height,
+                                   self.output_width))
+
+        return x
+
+
+class LocalizationNetwork(nn.Module):
+
+    def __init__(self, output_size, pretrained=True, affine=True):
+        super(LocalizationNetwork, self).__init__()
+
+        self.cnn = nn.Sequential(
+            nn.Conv2d(3, 16, kernel_size=3, stride=1, padding=1),
+            nn.ReLU(inplace=True),
+            nn.Conv2d(16, 16, kernel_size=3, stride=1, padding=1),
+            nn.ReLU(inplace=True),
+            nn.MaxPool2d(kernel_size=2, stride=2),
+            nn.Conv2d(16, 32, kernel_size=3, stride=1, padding=1),
+            nn.ReLU(inplace=True),
+            nn.Conv2d(32, 32, kernel_size=3, stride=1, padding=1),
+            nn.ReLU(inplace=True),
+            nn.MaxPool2d(kernel_size=2, stride=2),
+        )
+
+        self.affine = affine
+
+        if self.affine:
+            self.linear = nn.Linear(32 * 16 * 16, 6)
+            self.warper = WarpAffine(output_size)
+        else:
+            self.linear = nn.Linear(32 * 16 * 16, 9)
+            self.warper = WarpPerspective(output_size)
+
+    def forward(self, x):
+        bs = x.size(0)
```

Comment:
use meaningful and readable variable names: `bs -> batch_size`

Reply:
Agreed :+1: